docs/install.rst

```diff
@@ -3,24 +3,27 @@
 Installation
 ============
 
-This part of the documentation covers the installation of the dyn module.
-The first step to using any software package is getting it properly installed.
+This part of the documentation covers the installation of the Dyn module.
 
 
 Distribute & Pip
 ----------------
 
-Installing dyn is simple via `pip <http://www.pip-installer.org/>`_.
+The easiest way to install the Dyn module is via ‘pip’.
+
+1. Go to <http://www.pip-installer.org/>’, download and install ‘pip’.
```

Comment:
I'm generally of the opinion that if someone is on github looking at the source code, that we probably don't need to tell them to download and install pip, or that they should be using a Terminal application. That being said, I guess it's not the end of the world to add this, just that it might be out of place for the intended audience